apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -473,6 +438,74 @@ export abstract class BaseInstallManager {
     return { shrinkwrapIsUpToDate, variantIsUpToDate };
   }
 
+  /**
+   * Git hooks are only installed if the repo opts in by including files in /common/git-hooks
+   */
+  private _installGitHooks(): void {
+    const hookSource: string = path.join(this._rushConfiguration.commonFolder, 'git-hooks');
+    const git: Git = new Git(this.rushConfiguration);
+    const hookDestination: string | undefined = git.getHooksFolder();
+
+    if (FileSystem.exists(hookSource) && hookDestination) {
+      const allHookFilenames: string[] = FileSystem.readFolder(hookSource);
+      // Ignore the ".sample" file(s) in this folder.
+      const hookFilenames: string[] = allHookFilenames.filter((x) => !/\.sample$/.test(x));
+      if (hookFilenames.length > 0) {
+        console.log(os.EOL + colors.bold('Found files in the "common/git-hooks" folder.'));
+
+        if (!git.isHooksPathDefault()) {
+          const color: (str: string) => string = this.options.bypassPolicy ? colors.yellow : colors.red;
+          console.error(
+            color(
+              [
+                ' ',
+                `Rush cannot install the "common/git-hooks" scripts because your Git configuration specifies "core.hooksPath=${git.getConfigHooksPath()}".`,
+                'You can remove the setting by running:',
+                ' ',
+                '    git config --unset core.hooksPath',
+                ' '
+              ].join(os.EOL)
+            )
+          );
+          if (!this.options.bypassPolicy) {
```

Comment:
If the person chooses "--bypass-policy", we had specified that the Rush's hooks should NOT be installed into the .husky folder.  Whereas this code falls through.

I can fix that.

Reply:
Fixed ✔